jraft-core/src/test/java/com/alipay/sofa/jraft/storage/impl/BaseLogStorageTest.java

```diff
@@ -149,10 +149,10 @@ public void testAddManyEntries() {
         assertEquals(0, this.logStorage.getFirstLogIndex());
         assertEquals(9, this.logStorage.getLastLogIndex());
         for (int i = 0; i < 10; i++) {
-            assertEquals(i, this.logStorage.getTerm(i));
-            final LogEntry entry = this.logStorage.getEntry(i);
-            assertNotNull(entry);
-            assertEquals(entries.get(i), entry);
+            final LogEntry logEntry = this.logStorage.getEntry(i);
```

Comment:
In `BaseLogStorageTest`, there are many similar `getTerm` calls. Would you like to update them all at once?

Reply:
ok